packages/rspack/tests/Defaults.unittest.ts

```diff
@@ -445,99 +445,101 @@ describe("snapshots", () => {
-		-       "maxAsyncRequests": Infinity,
-		-       "maxInitialRequests": Infinity,
-		+       "enforceSizeThreshold": 50000,
-		+       "maxAsyncRequests": 30,
-		+       "maxInitialRequests": 30,
-		@@ ... @@
-		-       "minSize": 10000,
-		+       "minSize": 20000,
-		@@ ... @@
-		-       "hash": false,
-		+       "hash": true,
-		@@ ... @@
-		-       "hash": false,
-		+       "hash": true,
-	`)
+@@ ... @@
+-         "localIdentName": "[path][name][ext]__[local]",
++         "localIdentName": "[hash]",
+@@ ... @@
+-     "minifyOptions": undefined,
++     "minifyOptions": Object {
++       "dropConsole": false,
++       "extractComments": "true",
++       "passes": 1,
++       "pureFuncs": Array [],
++     },
+@@ ... @@
+-     "treeShaking": false,
++     "treeShaking": true,
+@@ ... @@
+-   "mode": "none",
++   "mode": undefined,
+@@ ... @@
+-     "minimize": false,
++     "minimize": true,
+@@ ... @@
+-     "moduleIds": "named",
++     "moduleIds": "deterministic",
+@@ ... @@
+-     "sideEffects": "flag",
++     "sideEffects": true,
+@@ ... @@
+-       "enforceSizeThreshold": 30000,
+-       "maxAsyncRequests": Infinity,
+-       "maxInitialRequests": Infinity,
++       "enforceSizeThreshold": 50000,
++       "maxAsyncRequests": 30,
++       "maxInitialRequests": 30,
+@@ ... @@
+-       "minSize": 10000,
++       "minSize": 20000,
+@@ ... @@
+-       "hash": false,
++       "hash": true,
+@@ ... @@
+-       "hash": false,
++       "hash": true,
+`)
 	);
 	test("production", { mode: "production" }, e =>
 		e.toMatchInlineSnapshot(`
-		- Expected
-		+ Received
+- Expected
++ Received
 
-		@@ ... @@
-		-         "localIdentName": "[path][name][ext]__[local]",
-		+         "localIdentName": "[hash]",
-		@@ ... @@
-		-     "minifyOptions": undefined,
-		+     "minifyOptions": Object {
-		+       "dropConsole": false,
-		+       "passes": 1,
-		+       "pureFuncs": Array [],
-		+     },
-		@@ ... @@
-		-     "treeShaking": false,
-		+     "treeShaking": true,
-		@@ ... @@
-		-   "mode": "none",
-		+   "mode": "production",
-		@@ ... @@
-		-     "minimize": false,
-		+     "minimize": true,
-		@@ ... @@
-		-     "moduleIds": "named",
-		+     "moduleIds": "deterministic",
-		@@ ... @@
-		-     "sideEffects": "flag",
-		+     "sideEffects": true,
-		@@ ... @@
-		-       "enforceSizeThreshold": 30000,
-		-       "maxAsyncRequests": Infinity,
-		-       "maxInitialRequests": Infinity,
-		+       "enforceSizeThreshold": 50000,
-		+       "maxAsyncRequests": 30,
-		+       "maxInitialRequests": 30,
-		@@ ... @@
-		-       "minSize": 10000,
-		+       "minSize": 20000,
-		@@ ... @@
-		-       "hash": false,
-		+       "hash": true,
-		@@ ... @@
-		-       "hash": false,
-		+       "hash": true,
-	`)
+@@ ... @@
+-         "localIdentName": "[path][name][ext]__[local]",
++         "localIdentName": "[hash]",
+@@ ... @@
+-     "minifyOptions": undefined,
++     "minifyOptions": Object {
++       "dropConsole": false,
++       "extractComments": "true",
```

Comment:
Should be false in production mode.

Reply:
Should always be false in production mode?

Reply:
Not sure, but it is a breaking change. Although webpack extractComments by default in `production` mode, need to discuss it.